schedule.md

```diff
@@ -26,7 +26,10 @@ This will allow us to have a less packed schedule and more time for you to enjoy
 
 ### Exact schedule
```

Comment:
Let's remove the whole block of text above this (and this heading).
Let's also add the links to the talks and workshop overview as we have that on the talks and workshop sites itself.